lib/elixir_boilerplate_web/health/plug.ex

```diff
@@ -0,0 +1,14 @@
+defmodule ElixirBoilerplateWeb.Health.Plug do
+  use Plug.Builder
+
+  import ElixirBoilerplate.Gettext
+
+  def call(%{request_path: "/health"} = conn, _) do
+    conn
+    |> put_resp_header("content-type", "text/html")
```

Comment:
It needs the HTML content type to succeed?

Reply:
I think we should actually enforces the practice of returning a serialized JSON object instead of HTML (or plain text content), what do you think?

Reply:
I don't think the payload format actually matters since the load balancer only validates the `200` response.

But to continue along the previous comments, we probably should expose _metadata_ instead of "ok" and not rely on `dgettext/2` since this will never have to be translated.

I think returning the version number might be handy down the road for operational purposes… Something along those lines:

```elixir
def call(%{request_path: "/health"} = conn, _) do
  conn
  |> json(%{status: "ok", version: Application.spec(:elixir_boilerplate, :vsn)})
  |> halt()
```

Reply:
We won’t be able to use `json/2` since it’s we’re in a simple Plug and not a Phoenix controller action but we get the idea 😁